src/edges_estimate/foregrounds.py

```diff
@@ -302,3 +302,44 @@ def calculate(self, ctx, **params):
                     break  # only add to one thing, otherwise it's doubling up.
                 else:
                     ctx[key] *= bias
+
+
+@attr.s
+class LogPoly(Foreground):
+    """
+    LogPoly model from Sims et.al. 2020 (Equation 18)
+    T_{Fg} = 10^sum(d_i*log10(ν/ν_0)^i)_{i=1 to N}
+    Parameters
+    ----------
+    poly_order : int
+        The maximum polynomial order will be `poly_order `. There are `poly_order`
```

Comment:
this isn't true given the below implementation... you need to do `range(1, self.poly_order + 1)` to get the maximum order as `self.poly_order`.